applications/Liberland_judiciary.md

```diff
@@ -0,0 +1,533 @@
+# Liberland Blockchain - the Judiciary
+
+- **Team Name:** Liberland Blockchain Team
+- **Payment Address:** Polkadot 13uikMrP8PvgapAhskyHgM6E8Yw7E6C9LxXMZBpgp2Dm8Nsi
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview :page_facing_up:
+
+Follow-up on Liberland Grant Application [pull request: Create liberland #831; a623bb28f2c9ce9609bf46e0e5801cda6e55aa47] 
+
+### Liberland Blockchain
+
+We are Liberland, the world's freest country. Currently, we are settling the physical land and developing our blockchain. 
+
+Liberland Blockchain is a sovereign blockchain founded and supported by Liberland which provides the first set of use cases. The codebase is a fork of Substrate/Polkadot, with MIT license. Liberland does not claim ownership of the blockchain or related materials.
+
+Functionalities such as trading and DeFi projects are permissionless by default while some critical functionalities such as country politics are permissioned and require KYC.
+
+The next blockchain milestone is translating traditional contract, dispute resolution and judiciary functionalities into the Web3 realm, synergizing the virtues of centralized and decentralized models.
+
+This milestone implements the quintessential element of all governance systems - the judiciary. We seek to enable dispute resolution within this blockchain ecosystem.
+
+### Overview
+
+#### Project Overview
+
+The Liberland Judiciary is developing an on-chain system capable of acting on behalf of a party without their explicit permission in pre-determined ways with mutual consent given in advance.
```

Comment:
In what ways does the judiciary act on behalf of another party? Can you give examples? This sounds like a fairly simple role-based setup akin to [AccessControl](https://github.com/Pendzl/pendzl/blob/main/contracts/src/access/access_control/access_control.rs).

Reply:
Question, should i answer your questions in comments or update the application to be more clear?

Reply:
The best way usually is to add info to the document and then post a summary as a PR comment. Thanks.

Reply:
I believe its explained in the rest of the proposal. Basically, the contracts define the dispute mode and actions available in dispute mode and the way to trigger them. Sometimes it can be simple proxy/access control, but dispute resolution can be a more complex process.

We will probably make use of some contract library like openbrush. Any reason you specifically suggested pendzl instead of openbrush?

Reply:
> Any reason you specifically suggested pendzl instead of openbrush?

Pendzl is the successor to OpenBrush, which is not being developed anymore.